src/utils.js

```diff
@@ -41,3 +43,26 @@ export const removeEmptyKeys = (obj) => {
 export function sleep(ms) {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
+
+export const updateFlagImage = (barcode, src) => {
+  return axios.put(
+    `https://amathjourney.com/api/off-annotation/flag-image/${barcode}`,
+    {
+      mode: "no-cors",
+      imgid: Number(src[src.length - 5]),
```

Comment:
would be useful to have somewhere a comment showing how the `src` looks like, to have an idea of why/how this code (`src[src.length - 5]`) works... also, is there any case where the number consist in more than 1 digit?

Reply:
For now I don't see where else it could be used.

About the `src[src.length - 5]` I added a comment about how it can be avoid. So it will be possible to pass both `src` and `imgid` to this function

Reply:
Yes, of course as soon as a product has more than 9 images, it uses 2 digits numbers 😉